modules/client/main.tf

```diff
@@ -24,19 +24,11 @@ locals {
   client_env_script = file(
     "${path.module}/templates/scripts/forseti-client/forseti_environment.sh.tpl",
   )
-  client_conf = file(
-    "${path.module}/templates/configs/forseti_conf_client.yaml.tpl",
-  )
-  client_conf_path   = "${var.forseti_home}/configs/forseti_conf_client.yaml"
-  client_sa_name     = "forseti-client-gcp-${var.suffix}"
-  client_name        = "forseti-client-vm-${var.suffix}"
-  client_bucket_name = "forseti-client-${var.suffix}"
-  client_zone        = "${var.client_region}-c"
-
-  client_project_roles = [
-    "roles/storage.objectViewer",
-    "roles/cloudtrace.agent",
-  ]
+
+  client_conf_path = "${var.forseti_home}/configs/forseti_conf_client.yaml"
+
+  client_name = "forseti-client-vm-${var.suffix}"
+  client_zone = "${var.client_region}-c"
```

Comment:
In the import script [here](https://github.com/forseti-security/terraform-google-forseti/blob/c73c874f833afb2759db08e09271707332b5c50f/helpers/import.sh#L35), it looks like users are using the entire zone name, which includes the `-c` or whatever they have appended. If the user's import does not have `-c` appended at the end, will this VM be destroyed and then recreated with a zone that has `-c` appended instead?

Reply:
This is true.  This is actually a problem regardless of submodules.  May need to put another input variable here and in the server.